pyproject.toml

```
@@ -108,5 +103,10 @@ module = [
 ]
 ignore_errors = true
 
+[tool.poetry.plugins."console_scripts"]
+"version-check" = "exasol.toolbox.pre_commit_hooks.package_version:main"
```

Could the name have possible conflicts?
Maybe better:
```suggestion
"exa-version-check" = "exasol.toolbox.pre_commit_hooks.package_version:main"
```

---

This actually did not change, it just shows up because i moved something out of the section. Projects and other parts of the code rely on "version-check" as symbol.